src/locales/en/translation.json

```diff
@@ -789,7 +789,13 @@
     "walletHistory": "Wallet must be connected.",
     "loading": "Loading..."
   },
+  "validationErrors": {
+    "insufficientBalance": "insufficient funds available",
```

Comment:
Make sure you have final texts.

Reply:
I still don't think those are the final texts, ask @t-z-u .

Reply:
I just talked to tzu and he said it's the final message.